scripts/qa_crowbarsetup.sh

```diff
@@ -568,6 +564,21 @@ function onadmin_prepare_sles12_repos()
     fi
 }
 
+function download_and_mount_sles()
+{
+    local iso_dir="$1"
+    local mountpoint="$2"
+
+    local iso_file=SLES-$slesversion-DVD-x86_64-$slesmilestone-DVD1.iso
+    local iso_path=$iso_dir/$iso_file
+
+    local url="http://$susedownload/install/SLES-$slesversion-$slesmilestone/$iso_file"
+    safely wget --progress=dot:mega -nc -O$iso_path "$url" \
+        || complain 72 "iso not found"
```

Comment:
Yes you're right - adding `safely` was not supposed to be part of this commit and must have crept in at some point due to a rebase gone wrong.  Will fix.